packages/edit-post/src/components/layout/style.scss

```diff
@@ -1,6 +1,67 @@
+$shadow: drop-shadow(0 -1px rgba($color: #000, $alpha: 0.133)); // 0.133 = $gray-200 but with alpha.
+
+.edit-post-layout__metaboxes-resizable-area {
+	padding-top: 23px;
+	filter: $shadow;
+	background-color: $white;
+
+	&:not(.has-user-size) {
+		max-height: 50% !important;
+	}
+
+	& .components-resizable-box__handle-top {
```

Comment:
Rather than overwriting the existing handle with CSS, as implemented [here](https://github.com/WordPress/gutenberg/blob/fd8a8d1aa7d6e1ac5c37d424bd28609ed66266cb/packages/edit-site/src/components/resizable-frame/index.js#L271), how about defining a custom handle?

If we use a button as the handle and add a keyDown event, we should be able to focus the resize handle with the keyboard, making it operable.

Reply:
I’d thought this too and even think it’d be necessary but hoped to spare some effort while uncertain if this PR will garner favor.

Reply:
Now that the PR is approved, would it be a good time to apply @t-hamano 's advice?

Reply:
I’ve got a branch that does it. It’s looking like about `160 insertions(+), 40 deletions(-)` minimum from this branch so not too trivial. There are also design/UX considerations so a clean thread would be nice for any discussion. I’m thinking a separate PR. We can either merge to this one or to trunk if we land this before then.

Reply:
Sounds good — feel free to ping on that separate PR